### DISCUSSION TITLE
Food

### DISCUSSION THREAD

**Discussion by IAmLime** [DiscordDungeons/Bugs#1017] [Ideas]
Crafting or cooking food for food buffs or like make a hunger bar which you need to fill to adv and travel.
Overall it's a just suggestion to make items you get from foraging and fishing usefull.

**Comment by mcjian9** [comment]
overall its a suggestion to make this more like minecraft

**Comment by Sven65** [comment]
We've had plans on a cooking system before, but we didn't get far with implementing it. We'll be revisiting it in the future.
